### PR TITLE
Rename user to account

### PR DESCRIPTION
**Pull request by CodeSandwich**

### CODE REVIEW

File: test/Drips.t.sol [hunk 1]
```diff
@@ -23,21 +23,19 @@ contract DripsTest is Test {
     IERC20 internal defaultErc20;
     IERC20 internal otherErc20;
 
-    // Keys are user ID and ERC-20
-    mapping(uint256 => mapping(IERC20 => StreamReceiver[])) internal currStreamsReceivers;
-    // Key is user IDs
-    mapping(uint256 => SplitsReceiver[]) internal currSplitsReceivers;
+    mapping(uint256 accoutnId => mapping(IERC20 => StreamReceiver[])) internal currStreamsReceivers;
```
Comment (xmxanuel) [inline test/Drips.t.sol:26]:
```suggestion
    mapping(uint256 accountId => mapping(IERC20 => StreamReceiver[])) internal currStreamsReceivers;
```

File: test/Drips.t.sol [hunk 2]
```diff
@@ -608,91 +611,91 @@ contract DripsTest is Test {
...
+        accountMetadata[0] = AccountMetadata("key", "value");
         vm.expectRevert(ERROR_NOT_DRIVER);
-        drips.emitUserMetadata(user, userMetadata);
+        drips.emitAccountMetadata(account, accountMetadata);
     }
 
     function testSetStreamsLimitsTotalBalance() public {
         uint128 splitsBalance = uint128(drips.MAX_TOTAL_BALANCE()) / 10;
-        give(user, receiver, splitsBalance);
+        give(account, receiver, splitsBalance);
```
Comment (xmxanuel) [inline test/Drips.t.sol:636]: In most other places it is called `accountId` but i guess it is fine

**Review by xmxanuel — APPROVED** [review]
looks good. found one typo

File: test/Drips.t.sol (hunk 2 above)
Comment (CodeSandwich) [reply test/Drips.t.sol]: Fixed

**Pull request merged**